packages/vhd-lib/src/vhd.js

```diff
@@ -336,11 +329,11 @@ export default class Vhd {
           `freeFirstBlockSpace: move first block ${firstSector} -> ${newFirstSector}`
         )
         // copy the first block at the end
-        const stream = await this._readStream(
+        const block = await this._read(
```

Comment:
Maybe we should keep streams when copying data and just change where we need to use buffers.

What do you think?

Reply:
I think the blocks are small enough (2MB) that it doesn't matter if we interleave read and write, but being the only use of stream, we gain some code by removing it.

Reply:
Fine by me :slightly_smiling_face: